docs/content_management/data_migration/importing_data.md

```diff
@@ -22,33 +22,50 @@ Migrations store execution metadata in the `ibexa_migrations` database table.
 This allows incremental upgrades:
 the `ibexa:migrations:migrate` command ignores files that it had previously executed.
 
+## Migration step
+
+A migration is a list of steps. A step is an action mode on a type of objects.
+
+In a migration file, a step is an array item starting with the mandatory properties `type` and `mode`, for example:
+
+```yaml
+-
+    type: content
+    mode: create
+```
+
+Then, the step is described by additional properties depending on its type and mode.
+
+* See [Available migrations](#available-migrations) for the modes available for each type.
+* See [Migration examples](#migration-examples) to explore what you can do with each type.
+
```

Comment:
```suggestion
* See [Migration examples](#migration-examples) to explore what you can do with each type.
* For a custom migration step, see [Create data migration step](create_data_migration_step.md).

```

TODO: There is a short definition of a migration step in [Create data migration step](https://ez-systems-developer-documentation--2175.com.readthedocs.build/en/2175/content_management/data_migration/create_data_migration_step/) that could be mutualized with this one.